frontend/src/book/DeleteBook.tsx

```diff
@@ -51,7 +51,16 @@ function DeleteBook(props: DeleteBookProps) {
                 <span className="modal-book-info">
                     {props.book.title}<br />
                     from <br />
-                    {props.book.author}
+                    {props.book.author}<br />
+                    {props.book.isbn.map(current => {
+                        return (
+                            <>
+                                <p key={current.identifier}>{current.type}: {current.identifier}</p>
+                            </>
+
+                        )
+                    })
```

Comment:
Den <>-Tag kannst Du hier auch weglassen, weil Du nur ein HTML-Element hast.

```suggestion
                    {props.book.isbn.map(current => {
                        return <p key={current.identifier}>{current.type}: {current.identifier}</p>
                    }
```

Reply:
stimmt, ich habe <> Tag rausgenommen :)